coding/html.md

```diff
@@ -9,3 +9,7 @@ General rules for all HTML documents are:
  - Tags and attributes should be lowercase (`<p class="intro">` not `<P CLASS="intro"`)
  - Use double quotes for attribute values
  - Close all elements - either a closeing tag (`<p>...</p>`) or self-closing (`<img src="jeff.jpg" />`)
+
+## Images
+  - Use alt attributes on img elements
```

Comment:
I would say use descriptive alt text about an image. Describe what the image is showing not what it is or filename. Can you also add some examples of dos and don'ts. Might help demo the differences.